platform/android/README.md

```diff
@@ -98,4 +98,4 @@ More information about building and distributing this project in [DISTRIBUTE.md]
 #### Symbolicating native crashes
 
 When hitting native crashes you can use ndk-stack to symbolicate crashes. 
-More information in [this](https://github.com/mapbox/mapbox-gl-native/wiki/Getting-line-numbers-from-an-Android-crash-with-ndk-stack) guide.
\ No newline at end of file
+More information in [this](https://github.com/mapbox/mapbox-gl-native/wiki/Getting-line-numbers-from-an-Android-crash-with-ndk-stack) guide.
```

Comment:
@zugaldia I didn't touch anything here so i'm not sure why/what changes being tracked are. Would love 👀  to make sure we're good to go.

Reply:
@Lndsay not sure either - everything looks fine and link is still working, let's 🚢 .